src/services/Content/ContentApi.ts

```diff
@@ -330,12 +330,33 @@ export const introspectRepository: (
   return data;
 };
 
+export const triggerSnapshot: (
+  repositoryUUID: string,
+  resetCount: number,
+) => Promise<void> = async (repositoryUUID) => {
+  try {
+    const { data } = await axios.post(
+      `/api/content_sources/v1.0/repositories/${repositoryUUID}/snapshot/`,
+      {
+        repository_uuid: repositoryUUID,
```

Comment:
the reason you're getting undefined is because the variable in the template string URL here is 'repositoryUUID', but here you're defining it as repository_uuid.  Change the variable in the URL to  ${repository_uuid}  and i imagine it'll start working

Reply:
apologies, the repositoryUUID is correct, the line `  repository_uuid: repositoryUUID,` is not needed at all.